cmd/ec2cmd.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import "fmt"
+
+func init() {
+	ec2Cmd := &cmd{
+		name:        "ec2",
+		help:        "Launch ec2 instances",
```

Comment:
Since we'll not only trigger launch, but terminate, list etc. them also:

```suggestion
		help:        "Manage ec2 instances",
```

Reply:
Makes sense.